src/main/resources/camunda/hearing_fee_check_scheduler.bpmn

```diff
@@ -10,7 +10,7 @@
     <bpmn:startEvent id="StartEvent_1">
       <bpmn:outgoing>Flow_03at42s</bpmn:outgoing>
       <bpmn:timerEventDefinition id="TimerEventDefinition_1oppfcm">
-        <bpmn:timeCycle xsi:type="bpmn:tFormalExpression">0 0/10 * * * ?</bpmn:timeCycle>
+        <bpmn:timeCycle xsi:type="bpmn:tFormalExpression">0 0/10 * * * ? 2024</bpmn:timeCycle>
```

Comment:
Is it required to run this expression for every 10 min in 2024? if not we should just make it once a day in future

Reply:
We will. This is just temporary to disable that cron job altogether to allow for the release.

A separate commit will address that 10 minutes schedule and turn it into a daily one.

Reply:
@mounikahmcts  this is an emergency fix to stop the scheduler with minimal change without impacting the release. The proper fix for this is on a PR which will be merged after the release